src/main/java/core/basesyntax/service/impl/ProcessServiceCsvImpl.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.constants.Activity;
+import core.basesyntax.constants.Product;
+import core.basesyntax.service.ProcessService;
+import core.basesyntax.service.strategy.ActivityTypeStrategy;
+import core.basesyntax.service.strategy.ActivityTypeStrategyImpl;
+import core.basesyntax.strategy.ActivitiesStrategy;
+import core.basesyntax.strategy.ActivitiesStrategyImpl;
+import core.basesyntax.strategy.handlers.ActivityHandler;
+import core.basesyntax.strategy.handlers.impl.BalanceActivityHandler;
+import core.basesyntax.strategy.handlers.impl.PurchaseActivityHandler;
+import core.basesyntax.strategy.handlers.impl.ReturnActivityHandler;
+import core.basesyntax.strategy.handlers.impl.SupplyActivityHandler;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public class ProcessServiceCsvImpl implements ProcessService {
+    private static final String APPLE_CSV_TYPE = "apple";
+    private static final String BANANA_CSV_TYPE = "banana";
+    private static final String CSV_PUNCTUATION_MARK = ",";
+    private static final Integer ACTIVITY_CSV_TYPE = 0;
+    private static final Integer PRODUCT_CSV_TYPE = 1;
+    private static final Integer AMOUNT_CSV_TYPE = 2;
+    private static final String BALANCE_CSV_TYPE = "b";
+    private static final String SUPPLY_CSV_TYPE = "s";
+    private static final String PURCHASE_CSV_TYPE = "p";
+    private static final String RETURN_CSV_TYPE = "r";
+    private static final Map<Activity, ActivityHandler> strategyMap = Map.of(
+            Activity.BALANCE, new BalanceActivityHandler(),
+            Activity.PURCHASE, new PurchaseActivityHandler(),
+            Activity.RETURN, new ReturnActivityHandler(),
+            Activity.SUPPLY, new SupplyActivityHandler());
+    private static final Map<String, Activity> strategyActivityTypeMap = Map.of(
+            BALANCE_CSV_TYPE, Activity.BALANCE,
+            SUPPLY_CSV_TYPE, Activity.SUPPLY,
+            PURCHASE_CSV_TYPE, Activity.PURCHASE,
+            RETURN_CSV_TYPE, Activity.RETURN);
```

Comment:
cannot get why we need 2 maps
```suggestion
    private static final Map<String, ActivityHandler> strategyMap = Map.of(
            BALANCE_CSV_TYPE, new BalanceActivityHandler(),
            PURCHASE_CSV_TYPE, new PurchaseActivityHandler(),
            RETURN_CSV_TYPE, new ReturnActivityHandler(),
            SUPPLY_CSV_TYPE, new SupplyActivityHandler());
```